lib/react.dart

```diff
@@ -1042,19 +1030,8 @@ class SyntheticClipboardEvent extends SyntheticEvent {
     num timeStamp,
     String type,
     this.clipboardData,
-  ) : super(
-            bubbles,
-            cancelable,
-            currentTarget,
-            defaultPrevented,
-            preventDefault,
-            stopPropagation,
-            eventPhase,
-            isTrusted,
-            nativeEvent,
-            target,
-            timeStamp,
-            type) {}
+  ) : super(bubbles, cancelable, currentTarget, defaultPrevented, preventDefault, stopPropagation, eventPhase,
+            isTrusted, nativeEvent, target, timeStamp, type) {}
```

Comment:
Oh noooo is there anything we can do here?

Reply:
I'm actually fine with this, no use taking up a ton more lines 🤷‍♂️

Reply:
But yes, this could be forced onto multiple lines with trailing commas.